public/js/saveApplicant.js

```diff
@@ -32,26 +67,28 @@ let makeApiRequest = async (data) => {
         },
         method: 'post',
         body: JSON.stringify(data)
-    })
-        .then(response => response.json())
-        .then((data) => {
-            if (data.success) {
-                window.location.href = './admin';
-            } else {
-                document.getElementById('errorMsg').innerHTML = "Please contact administrator. Or <a href='./admin'>click here</a> to return to admin page"
-            }
-        })
+    }).then(response => {
+        if (response.status == 200) {
+            window.location.href = './admin';
+        } else if (response.status == 400) {
+            document.querySelector('#generalError').innerHTML = "You must fill out all form options.";
+            document.querySelector('#generalError').classList.remove('hidden');
+        } else {
+            document.querySelector('#generalError').innerHTML = "Something went wrong, please try again later.";
+            document.querySelector('#generalError').classList.remove('hidden');
```

Comment:
```suggestion
    }).then(response => {
     let ge = document.querySelector('#generalError');
        if (response.status == 200) {
            window.location.href = './admin';
        } else if (response.status == 400) {
            ge.innerHTML = "You must fill out all form options.";
            ge.classList.remove('hidden');
        } else {
            ge.innerHTML = "Something went wrong, please try again later.";
            ge.classList.remove('hidden');
```
more DRY. not perfect but smth like that